hack/make/binary

```diff
@@ -19,6 +19,16 @@ if [ "$(go env GOOS)/$(go env GOARCH)" != "$(go env GOHOSTOS)/$(go env GOHOSTARC
 	esac
 fi
 
+if [ "$(go env GOHOSTOS)/$(go env GOHOSTARCH)" == "windows/amd64" ] && [ "$(go env GOOS)" == "windows" ]; then
+	# native compilation of Windows on Windows with golang 1.5+ needs linkmode internal
+	# https://github.com/golang/go/issues/13070
+	if [[ $LDFLAGS_STATIC_DOCKER == *"-linkmode" ]]; then
+		export LDFLAGS_STATIC_DOCKER="${LDFLAGS_STATIC_DOCKER/-linkmode external/-linkmode internal}"
+	else
+		export LDFLAGS_STATIC_DOCKER="$LDFLAGS_STATIC_DOCKER -linkmode=internal"
```

Comment:
As it happens, at least with the current rest of the hack/* scripts, on Windows linkmode is never internal anyway. Maybe I should dispense with the replace, and just hard-code the `export LDFLAGS_STATIC_DOCKER="$LDFLAGS_STATIC_DOCKER -linkmode=internal"` line?